### DISCUSSION TITLE
如何配置文章阅读量？

### DISCUSSION THREAD

**Discussion by techkoala** [HEIGE-PCloud/DoIt#94] [Q&A] [answered]
![屏幕截图 2021-06-04 170909](https://user-images.githubusercontent.com/67769327/120777165-97878600-c557-11eb-861d-fd287a33f29e.jpg)
这个里面的30次阅读是在哪里配置呢？我找遍了也没找到这个选项，我的网站除了这个日期字数还有阅读时间都有。
刚开始以为时DoIt新加的功能，结果发现LoveIT原本就有，一直没注意
请问有人知道在哪儿设置吗？感谢

**Comment by HEIGE-PCloud** [comment, marked as answer]
https://hugodoit.pages.dev/zh-cn/theme-documentation-basics/#site-configuration

在网站配置中使用 [Valine](https://valine.js.org) 或者 [Waline](https://waline.js.org) 并开启 `visitor=true` 的选项。

**Reply by techkoala** [reply]
> 
> 
> https://hugodoit.pages.dev/zh-cn/theme-documentation-basics/#site-configuration
> 
> 在网站配置中使用 [Valine](https://valine.js.org) 或者 [Waline](https://waline.js.org) 并开启 `visitor=true` 的选项。

原来是这里的设置

感谢~

我去尝试一下
